tests/Models/Chat/ChannelTest.php

```diff
@@ -49,7 +49,7 @@ public function testChannelCanMessageModeratedPmChannel(?string $group, bool $ca
     {
         $user = User::factory()->withGroup($group)->create();
         $otherUser = User::factory()->create();
-        $channel = $this->createChannel([$user, $otherUser], 'moderated', 'pm');
+        $channel = $this->createChannel([$user, $otherUser], 'pm', 'moderated');
```

Comment:
the new signature for the third parameter is boolean, not string (not enforced here because no strict is declared and can be internally casted)